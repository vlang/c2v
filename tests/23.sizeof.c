#include <unistd.h>
int main() {
  char hello[] = "Hello, unistd world!\n";
  write ( 1, hello, sizeof hello );
  return 0; }