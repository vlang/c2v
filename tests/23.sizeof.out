@[translated]
module main

fn main() {
	hello := c'Hello, unistd world!\n'
	C.write(1, hello, sizeof(hello))
	return
}
